features/checkout_details.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Check Out
+  If user click to checkout button when cart is empty
+  Return homepage
```

Comment:
Should add more description here, for example:

If user click checkout button when cart is empty
I will be redirected to the homepage